diff --git a/src/extractPDF.js b/src/extractPDF.js
index 576e8956f5984b7790c7d07374ae856be5ddf242..d0b80bfcc99423ad01725f96becb2c117b2e2df9 100644
--- a/src/extractPDF.js
+++ b/src/extractPDF.js
@@ -23,7 +23,7 @@ async function extractPDF(url, baseUrl) {
 
   try {
     const res = await fetch(new URL("pdf/pdf.min.js", baseUrl).href);
-    eval(await res.text());
+    //eval(await res.text());
     
     //pdfjsLib should now exist
     // eslint-disable-next-line no-undef
